dronesym-node/index.js

```diff
@@ -57,7 +59,7 @@ app.use(function(err, req, res, next) {
 
 
 http.listen(3000, function(){
```

Comment:
I think you have to use your new `port` variable here also if you really want it to be used